README.md

```diff
@@ -1 +1,69 @@
-# todoWebpack_JS
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do list
+
+This project is about to give us a handle as a student on how to make interactive a web site.
+
+## Built With
+
+- Html, CSS , JavaScript & Webpack
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+If you want to use this project locally on your device you will need the following tools:
+    1: Node.js
+    2: NPM
+    3: Git
+
+### Setup
+
+ To get the project in your local environment, run 
+
+ ```
+
+ git clone git@github.com:GhislainMitahi/todoWebpack_JS.git
+ 
+ ```
+  in your terminal
+
+  To get actions run ```npm install```
+
+  
+
+
+
+## Authors
+
+👤 **Habib Payenda**
+
+- GitHub: [@HabibPayenda](https://github.com/githubhandle)
+- Twitter: [@HabibPayenda](https://twitter.com/twitterhandle)
+- LinkedIn: [@HabibPayenda](https://linkedin.com/in/linkedinhandle)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The link to the MIT license is broken because the file is missing in your project. Kindly add it 👍🏼